### PR TITLE
Issue #1937 Mapping in React Solved

### LINKED ISSUE #1937: Front End with Framework:  Mapping data to components in React
### Description

- How to map data into Components using ReactJS?
- Add examples (Must)

### Domain

Frontend Dev React/Angular/Vue

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Ajinkyap331**
<hr>

## Description 📜

FIxed Issue #1937 Mapping in React as a source of Documentation
Fixes #1937 
<hr>

## Type of change 📝

- [x] Doc#umentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev #React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

<hr>

### CODE REVIEW

**Review by heet-kumar — CHANGES_REQUESTED** [review]
Add References from where you have referred and give them credits for these documentation .

**Comment by Ajinkyap331** [comment]
React offical website

**Comment by heet-kumar** [comment]
> React offical website @heet-kumar

@Ajinkyap331 this is the not the way to specify you need to specify in document in proper manner at least specify 3 references .
that I also know that React official website and React official documentation is used . 
It is also mentioned in server-announcement channel of the GWOC server in Discord that in Every Pull Request you need to add References and give credits .

**Comment by Ajinkyap331** [comment]
Sorry @heet-kumar 
I will add atleast 3 of them by tonight

**Comment by heet-kumar** [comment]
> ## Description 📜
> Please include a summary of the change and which issue is fixed. Please also include relevant motivation and context.
> Fixes # <issue_number_here>
> 
> ## Type of change 📝
> * [x]  Doc#umentation (Content Creation in the form of codes or tutorials)
> 
> ## Domain of Contribution 📊
> * [x]  Frontend Dev #React/Angular/Vue
> 
> ## Checklist ✅
> * [x]  I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
> * [x]  I have performed a self-review of my own code or work.
> * [x]  I have commented my code, particularly in hard-to-understand areas.
> * [x]  My changes generates no new warnings.
> * [x]  I'm GWOC'21 contributor
> 
> ## Screenshots / Gif (Optional) 📸

Also mention Description here properly including issue number properly.

**Comment by Ajinkyap331** [comment]
@heet-kumar I have Done please check it once. I edited the comment above so please check if the description is added

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md:101]: instead of showing image just create table for showing output in markdown you can refer #2107 this gay made table for showing difference it look good .

Everything else is alright. If possible then try to do that.

**Review by heet-kumar — CHANGES_REQUESTED** [review]
make this change because table will be better than image .

**Review by heet-kumar — COMMENTED** [review]
Everything is alright ! Let us wait for the approval of supervisors and if they suggest any changes you can do that too! make that change and mention me here so that I call supervisor for merging .

**Comment by Ajinkyap331** [comment]
BTW I need to search it how to do in markdown 😂😂@heet-kumar 
I will check and edit it by evening

**Comment by heet-kumar** [comment]
> BTW I need to search it how to do in markdown 😂😂@heet-kumar
> I will check and edit it by evening

sure once completed then mention me here so that I can make a call for supervisor to review.

**Comment by Ajinkyap331** [comment]
@heet-kumar I have tried to add a table please check it once

**Comment by Navneetkumarjpr** [comment]
i want to do this assignment

**Comment by Ajinkyap331** [comment]
@Navneetkumarjpr ho gaya bro 🤣

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md:126]: create hyperlink on the name of Pluralsight like above one

**Review by heet-kumar — CHANGES_REQUESTED** [review]
just make this changes

File: Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md
Comment (heet-kumar) [inline Frontend_Web_Development_React_Angular_Vue/React/Week 2/Mapping_To_Components.md:106]: above this line just give heading of # Output

**Review by heet-kumar — CHANGES_REQUESTED** [review]
as you are showing output

**Review by heet-kumar — CHANGES_REQUESTED** [review]
just remove Mapping.jpeg image from the assets folder there is no use i think so

**Review by heet-kumar — APPROVED** [review]
that's all Lets Get Merge this PR !  let us wait for the approval of supervisors and if they suggest any changes you can do that too! so here I am giving approval from my side just don't forget to make changes which I have mentioned above.

**Comment by heet-kumar** [comment]
> i want to do this assignment

sorry buddy but @Ajinkyap331 is working quite good on this issue and it is already assigned .

**Comment by heet-kumar** [comment]
> @heet-kumar I have tried to add a table please check it once

damn good work

**Comment by heet-kumar** [comment]
@anandhu720 @the-pro review this PR once from your side and get merge.

**Review by anandhu720 — APPROVED** [review]

**Comment by Ajinkyap331** [comment]
Thanks, @heet-kumar  @anandhu720

**Pull request merged**
